src/utils/config.js

```diff
@@ -85,13 +85,42 @@ export const mergeConfig = (config) => {
   storeConfig(_.merge({}, currentConfig, config));
 };
 
-export const loadYargsConfig = () => {
-  const {
-    configuration: { profiles: _, ...configuration },
-    credentials: { profiles: __, ...credentials },
-  } = loadConfig();
-  return {
+export const loadProfileConfig = (profile) => {
+  const { configuration, credentials } = loadConfig();
+  const profileConfig = {
     ...configuration,
+    ...(profile && configuration.profiles && configuration.profiles[profile]),
     ...credentials,
+    ...(profile && credentials.profiles && credentials.profiles[profile]),
   };
```

Comment:
This type of merging will not handle undefined values. Is it okay?

Reply:
Can you reiterate? Spreading variable with `undefined` value should work just fine.